dist/index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link
+            href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+            rel="stylesheet"
+            integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC"
+            crossorigin="anonymous">
+        <link rel="stylesheet"
+            href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200"
+            />
+        <title>Document</title>
+    <script defer src="index.fbe9bb43.js"></script><script defer src="crud.fbe9bb43.js"></script></head>
+    <body>
+        <section class="container1">
```

Comment:
Kindly change this section tag to main instead.

“ It is not recommended to nest section tags inside one another because it can lead to confusion about the structure of the content and make it harder for search engines and assistive technologies to understand the hierarchy of the information on the page“